mmedit/datasets/sr_vid4_dataset.py

```diff
@@ -45,6 +53,7 @@ def __init__(self,
                  pipeline,
                  scale,
                  filename_tmpl='{:08d}',
+                 metric_average_mode='folder',
```

Comment:
What does the frames in a "folder" represent? A video/clip? If so, then the name should be "video" or "clip".
The metric should be independent of its data representation. Say if we implement an evaluation with raw video input, then the "folder" will be strange by then.